internal/monitoring/prometheus/metrics.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package prometheus contains bindings to prometheus for the interfaces in
+// the parent monitoring package.
```

Comment:
Are these files forked from the Trillian `monitoring/...` packages?

If so, it might be worth mentioning that somewhere in case there's some future archeology or refactoring work which comes along.

Reply:
Added a comment to this effect.